tests/FiveOhThreeTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Tests;
+
+
+use Mesavolt\FiveOhThree\LockFileException;
+use Mesavolt\FiveOhThree\LockGuard;
+use PHPUnit\Framework\TestCase;
+
+class FiveOhThreeTest extends TestCase
+{
+    protected function tearDown()
+    {
+        parent::tearDown();
+
+        // ensure each test gets a clean env
+        unset($_ENV['FIVEOHTHREE_IGNORE_LOCK']);
+    }
+
+    public function test_lockFileExists()
+    {
+        $this->assertTrue(LockGuard::lockFileExists(__FILE__), 'Lock file should be detected');
+        $this->assertFalse(LockGuard::lockFileExists('/i-do-not-exists'), "Lock file shouldn't be detected");
+
+        $_ENV['FIVEOHTHREE_IGNORE_LOCK'] = 'true';
+        $this->assertFalse(LockGuard::lockFileExists('/i-do-not-exists'), 'Lock file should be ignored');
```

Comment:
You're not testing anything here - you should ensure that checking an existing lock file existence returns false if the env var exists, isn't it?

```suggestion
        $this->assertFalse(LockGuard::lockFileExists(__FILE__), 'Lock file should be ignored');
```

Reply:
you right, i dumb